pkg/objstore/s3/s3.go

```diff
@@ -193,7 +205,11 @@ func (b *Bucket) Exists(ctx context.Context, name string) (bool, error) {
 // Upload the contents of the reader as an object into the bucket.
 func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader) error {
 	b.opsTotal.WithLabelValues(opObjectInsert).Inc()
-	_, err := b.client.PutObjectWithContext(ctx, b.bucket, name, r, -1, minio.PutObjectOptions{})
+
+	_, err := b.client.PutObjectWithContext(ctx, b.bucket, name, r, -1,
```

Comment:
somehow now it works with -1? ^^

Reply:
Yeah, I think it was access denied error manifesting in weird way